### ISSUE TITLE
runservo.sh script is incorrect in nightly build

### ISSUE THREAD

**Issue by NiuZhuang** [servo/servo#13977]
Issue: got 404 not found after run ./runservo.sh due to wrong index path parameter in script

Original: 

``` shell
#!/usr/bin/env sh
./servo -b --pref dom.mozbrowser.enabled --pref dom.forcetouch.enabled --pref shell.builtin-key-shortcuts.enabled=false ./build/browserhtml-5c846c4e388d17ec/out/index.html
```

Fixed:

``` shell
#!/usr/bin/env sh
./servo -b --pref dom.mozbrowser.enabled --pref dom.forcetouch.enabled --pref shell.builtin-key-shortcuts.enabled=false ./build/out/index.html
```

**Comment by aneeshusa** [comment]
Thanks for the report! This was introduced about a month ago but should be fixed as of #13558, which merged a few days ago. Please try the newest nightly.

**Comment by aneeshusa** [comment]
Closing since this has been fixed; please comment if this happens again.
